main.py

```diff
@@ -37,10 +38,10 @@ def reEmail(fname):
 def reAward(fname):
   fhand = open(fname, 'r')
   for line in fhand:
-    extr = re.findall('.*', line)
+    extr = re.findall('-.*', line)
```

Comment:
Your regex will extract all in front of the award winner.  "-  Usain Bolt"
extr = re.findall('- (.*)', line)